### DISCUSSION TITLE
delete file

### DISCUSSION THREAD

**Discussion by tomagotcha** [sumatrapdfreader/sumatrapdf#2804] [Ideas]
Hello, I don't know if it has been suggested before, but I think a button/function in SumatraPDF to delete the active pdf on the hard drive would be very useful. Thank you.

**Comment by GitHubRulesOK** [comment]
see https://github.com/sumatrapdfreader/sumatrapdf/issues/280 and https://github.com/sumatrapdfreader/sumatrapdf/issues/633#
 although deleting a file is an Operating System function there are problems with shooting self in foot since file can be in use not just by SumatraPDF think AntiVirus, Windows Search or 3rd party indexing or File Syncing or another previewer, 
current status see https://github.com/sumatrapdfreader/sumatrapdf/issues/2246

**Comment by tomagotcha** [comment]
Thank you, understood!

**Reply by GitHubRulesOK** [reply]
see the workaround in **[633 link](https://github.com/sumatrapdfreader/sumatrapdf/issues/633#issuecomment-778718892)** above it has worked for me when needing to process lots of deletions after review
